docs/src/templates/includes/components/core/je-previousOrderCard.hbs

```diff
@@ -0,0 +1,25 @@
+<a href="{{ restaurantLink }}"
+    class="c-previousOrderCard {{ optionalClassNames }}"
+    href="{{ restaurantLink }}"
+    title="{{ orderAgainLinkTitle }}"
+    data-gtm="home|click-previous|reorder"
+>
+
+    <div class="c-previousOrderCard-image">
+        <img src="{{ restaurantImageUrl }}" alt="" class="c-previousOrderCard-img">
```

Comment:
Should we have some text in the alt tag?

Reply:
✅